src/OrthogonalSphericalShellGrids/distributed_tripolar_grid.jl

```diff
@@ -265,15 +265,15 @@ end
 
 # Extension of the constructor for a `Field` on a `TRG` grid. We assumes that the north boundary is a zipper
 # with a sign that depends on the location of the field (revert the value of the halos if on edges, keep it if on nodes or centers)
-function Field((LX, LY, LZ)::Tuple, grid::DistributedTripolarGridOfSomeKind, data, old_bcs, indices::Tuple, op, status)
+function Field(loc::Tuple{<:Any, <:Any, <:Any}, grid::DistributedTripolarGridOfSomeKind, data, old_bcs, indices::Tuple, op, status)
```

Comment:
```suggestion
function Field(loc::Tuple{<:LX, <:LY, <:LZ}, grid::DistributedTripolarGridOfSomeKind, data, old_bcs, indices::Tuple, op, status) where {LX, LY, LZ}
```

Reply:
damn... I was working locally and then I couldn't push it... because you were faster than me!